packages/build/src/run-publish.ts

```diff
@@ -1,17 +1,19 @@
 import type { writeBuildInfo as writeBuildInfoType } from './build-info';
-import { Barque } from './barque';
+import type { Barque } from './barque';
+import type {
+  Config} from './config';
```

Comment:
Interesting use of whitespace :)

Reply:
Yeah, no big deal, prettier will take care of it, but it does just look a bit odd ;)